ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5193,3 +5193,75 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: bulkJobProcessCreate
+    request_path: "{{ program_service_prefix }}/v1/process/create"
+    upstream_url: "{{ program_service_url }}/program/v1/process/create"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: bulkJobProcessRead
+    request_path: "{{ program_service_prefix }}/v1/process/read"
+    upstream_url: "{{ program_service_url }}/program/v1/process/read"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: bulkJobProcessUpdate
+    request_path: "{{ program_service_prefix }}/v1/process/update"
+    upstream_url: "{{ program_service_url }}/program/v1/process/update"
+    strip_request_path: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'contentCreate'
```

Comment:
is this an admin functionality or any user can perform this update?

Reply:
Any user can perform this.

Reply:
looks good @venkateshwarans